src/main/java/lex/tasks/Deadline.java

```diff
@@ -0,0 +1,48 @@
+package lex.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+    protected LocalDate end;
```

Comment:
Maybe you could change the variable name into 'deadline' or 'endTime' for better understanding?